src/System.Reflection.Metadata/src/System/Reflection/Metadata/Ecma335/MetadataSizes.cs

```diff
@@ -339,7 +339,7 @@ public int MetadataHeaderSize
                     MetadataVersionPaddedLength +  // metadata version
                     sizeof(ushort) +               // storage header: reserved
                     sizeof(ushort) +               // stream count
-                    (IsStandaloneDebugMetadata ? StandalonePdbStreamHeaderSize : 0) + 
+                    (IsStandaloneDebugMetadata ? StandalonePdbStreamHeaderSize : 0) +
```

Comment:
Nit: There is only whitespace diff here. Would make the history easier to review if it were not in the same commit as the semantic changes.